components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/inbound/IdentityRequest.java

```diff
@@ -283,6 +285,31 @@ public IdentityRequestBuilder addParameters(Map<String, String[]> parameters) {
             return this;
         }
 
+        public IdentityRequestBuilder setAttributes(Map<String, Object> attributes) {
+            this.attributes = new HashMap(attributes);;
```

Comment:
We need to revert this back and need to have HashMap to copy the original hashmap from servlet request to the identity request